### ISSUE TITLE
Consider removing this forked repo

### ISSUE THREAD

**Issue by jslee02** [personalrobotics/dart#1]
I believe we forked this repo from `dartsim/dart` to create a temporary release `prl-v0.1.x` for Aikido. We needed a stable release that contains several new changes (at that moment) of DART that wasn't released yet. As Aikido now relies on DART 6.1, which is an officially released version, I think we don't need this repo anymore.

**Comment by mkoval** [comment]
I agree. I am happy to remove this repository if @psigen has no objections.

**Comment by psigen** [comment]
Sounds fine to me, please go ahead.

**Comment by jslee02** [comment]
Thanks! Deleting this repo.
